git_machete/cmd.py

```diff
@@ -3469,7 +3468,15 @@ def short_usage() -> None:
 
 
 def version() -> None:
-    print(f"git-machete version {__version__}")
+    try:
+        site_packages_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
```

Comment:
Wrt. naming of the variable: IIRC it'll land under `dist_packages` rather than `site_packages` when installed e.g. via `pip install`, pls check